### DISCUSSION TITLE
Does Zitadel support multi-region with Postgres?

### DISCUSSION THREAD

**Discussion by AndyScottt** [zitadel/zitadel#7636] [Q&A]
The documentation for multi-region is clear but seems specific to Cockoach, so I'm just curious if Cockroach is a requirement, or is Postgres can be used to achieve a multi-region Zitadel deployment, for example using RDS Postgres with cross-region read replicas enabled.

Thanks!

**Comment by fforootd** [comment]
At the moment we do not (yet) support the usage of read replicas because we want to keep most data consistent.

So what we usually recommend for Postgres is to deploy a multi-zonal single region cluster and have a cross-region replication for disaster recovery purpose.

With CockroachDB/Spanner et. al. cross region active-active deployments work. You might sacrifice a little on the latency side if your regions are far apart though.
